servers/rendering/renderer_rd/shaders/volumetric_fog.glsl

```diff
@@ -170,13 +170,13 @@ void main() {
 	if (params.shape == 0) {
 		//Ellipsoid
 		// https://www.shadertoy.com/view/tdS3DG
-		float k0 = length(local_pos.xyz / params.extents);
-		float k1 = length(local_pos.xyz / (params.extents * params.extents));
+		float k0 = length(local_pos.xyz / (params.size / 2.0));
+		float k1 = length(local_pos.xyz / (params.size / 2.0 * params.size / 2.0));
```

Comment:
```suggestion
		float k0 = length(local_pos.xyz / (params.size / 2.0));
		float k1 = length(local_pos.xyz / (params.size * params.size / 4.0));
```

Alternatively, a slightly more optimized choice would be to store extents above so you avoid doing this division multiple times

Reply:
You're right, I've added that now. I initially didn't optimise in this way as I was unsure about the approach to this kind of thing in in Godot, and whether that would be considered less clear (the programmer would need to work out that `params.size * params.size / 4.0` means `params.size / 2.0 * params.size / 2.0`, for example). I do agree with these changes though.